src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -283,6 +288,40 @@ static boolean hasOcr(CaseDetails theCase) {
         return valid(classification);
     }
 
+    public static Validation<String, Classification> hasJourneyClassificationForAttachToCase(CaseDetails theCase) {
+        Optional<String> classificationOption = getJourneyClassification(theCase);
+
+        return classificationOption
+            .map(classification -> Try.of(() -> Classification.valueOf(classification)))
+            .map(Try::toValidation)
+            .map(validation -> validation
+                .mapError(throwable -> "Invalid journeyClassification. Error: " + throwable.getMessage())
```

Comment:
If I understand correctly, this message will be returned to the caseworker. If so, it needs to be reduced to something like this: 
```
Invalid Journey Classification: {actual value}
```

No exception record message should be included